pod-batch.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: v1
+kind: Pod
+metadata:
+# this value should be differernt when running script again, as each time a script is run a new Pod is created
+  name: pod1
```

Comment:
That's the idea the the script will be manually updated on demand for a course site. So the `COURSE_ID` and `CATEGORY_NAME` will be different every time they run it. I don't think it should matter changing the POD name manually.